packages/linuxheaders.rb

```diff
@@ -3,23 +3,10 @@
 class Linuxheaders < Package
   description 'Linux headers for Chrome OS.'
   homepage 'https://kernel.org/'
-  version '3.18'
+  version '4.14'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
```

Comment:
This will require binaries before it's merged.

Reply:
I was able to build linuxheaders after a fresh install and the binaries have been pushed to this PR.  Since the linux_sources package is targeting release 75, I was unable to build linuxheaders for i686 so I just restored the 3.18 headers.  You do realize new binaries will need to be generated for each new release of ChromeOS?

Reply:
Can you please test the binaries on x86_64 when you get a chance?

Reply:
@uberhacker It will only need an update when Chrome OS updates their linux-headers version.

Reply:
It looks as if i686 is using the 4.4 Linux headers.

Reply:
@uberhacker LGTM

Reply:
Wow, this actually works to prevent sommelier from crashing after entering sleep mode.